src/coff_import_file.rs

```diff
@@ -128,6 +135,26 @@ fn set_name_to_string_table_entry(symbol: &mut ImageSymbol, offset: usize) {
     symbol.name[4..].copy_from_slice(&u32::try_from(offset).unwrap().to_le_bytes());
 }
 
+fn apply_name_type(import_type: ImportNameType, name: &str) -> &str {
+    fn ltrim1<'a>(name: &'a str, chars: &str) -> &'a str {
+        if let Some((first_char, rest)) = name.split_at_checked(1) {
+            if chars.contains(first_char) {
```

Comment:
```suggestion
        if let Some((first_char, rest)) = name.split_at_checked(1) && chars.contains(first_char) {
```

I think this would be a bit nicer with let chains.

Reply:
Sure: although that means moving to 2024 edition.

Reply:
Looks like the move to 2024 caused rustfmt to rearrange a bunch of code.